deployments/liqo/subcharts/discoveryOperator/templates/discovery-deploy.yaml

```diff
@@ -9,6 +9,148 @@ metadata:
     app: liqo.io
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: liqo-discovery-role
+rules:
+  - apiGroups:
+      - discovery.liqo.io
+    resources:
+      - foreignclusters
+      - searchdomains
+    verbs:
+      - get
+      - list
+      - patch
+      - update
+      - watch
+      - create
+      - delete
+  - apiGroups:
+      - discovery.liqo.io
+    resources:
+      - peeringrequests
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - config.liqo.io
+    resources:
+      - clusterconfigs
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+  - apiGroups:
+      - net.liqo.io
+    resources:
+      - networkconfigs
+      - tunnelendpoints
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - sharing.liqo.io
+    resources:
+      - advertisements
+    verbs:
+      - get
+      - list
+      - watch
+      - delete
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+    verbs:
+      - list
+  - apiGroups:
+      - rbac.authorization.k8s.io
+    resources:
+      - clusterroles
+    verbs:
+      - get
+      - create
+      - update
+  - apiGroups:
+      - rbac.authorization.k8s.io
+    resources:
+      - clusterrolebindings
+    verbs:
+      - get
+      - create
```

Comment:
Just curiosity, why do you need this? You are effectively becoming `cluster-admin`

Reply:
The role of the crdReplicator is created dynamically when a peering occurs, this is required to do that